src/cts/test/max_cap.tcl

```diff
@@ -4,7 +4,15 @@ read_lef sky130hs/sky130hs.tlef
 read_lef sky130hs/sky130hs_std_cell.lef
 read_liberty sky130hs/sky130hs_tt.lib
 
-set def_file "max_cap.def"
+if {[info exists ::env(TEST_TMPDIR)] && [llength $::env(TEST_TMPDIR)] > 0} {
+  # Bazel tmp dir per test
+  set tmpdir [lindex $::env(TEST_TMPDIR) 0]
+} else {
+  # Same as before Bazel
+  set tmpdir .
```

Comment:
Probably ok for now (as it was like that before), but of course storing things out-of-tree should maybe be a longer-term goal.

Reply:
Letting that sleeping dog lie, not a concern of this PR

Reply:
Actually, Bazel [linux-sandbox](https://bazel.build/docs/sandboxing) remaps /tmp, so using mktemp would have been fine. The above code is fine too, even if it is somewhat Bazel specific

Reply:
In the cmake case it should be writing to the results directory but that's not an issue for this PR.